blocks/library/button/index.js

```diff
@@ -130,6 +119,18 @@ registerBlockType( 'core/button', {
 					</InspectorControls>
 				}
 			</span>,
+			focus && (
+				<form
+					className="blocks-format-toolbar__inline-link-modal"
```

Comment:
I think we do not want to share the style of this class with other blocks, it may make sense to rename the classname to match our guidelines. Something like `blocks-button__inline-link`. Ans style it as part of the `editor.scss` of the button block